_topics/en/adaptor-signatures.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Adaptor signatures
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Privacy Enhancements
+  - Scripts and Addresses
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Adaptor signatures** are auxiliary signature data that commit to a a
+  hidden value.  When an adaptor signature is combined with a final
```

Comment:
Maybe another sentence for this intro section and motivation? Something like: "These are partial signatures that, if completed, will reveal valuable information to the signer. (In practice | For example ), when ..." (taken from abstract of [here](https://pdfs.semanticscholar.org/fdef/41889571e22f86e95a6eb1838aaa06bb5737.pdf))

Reply:
To be honest, I think the term "final signature" is not necessary. We have a term for this thing, namely "digital signature". 

Tbh, I think the term "adaptor signature" is misleading, it's rather a signature adaptor. But I guess we can't change this. That being said, I think it's clever to avoid the plain term "signature" below, unless it's really clear from the context. I think it should always be "adaptor signature" or "digital signature" (or "Schnorr signature"/"BIP340 signature")

Reply:
Yes, I've also multiple times expressed concern that "adaptor signature" will lead to people thinking of it as in some way a signature, when it is about as far from a signature as possible, considering it's trivially unsound/forgeable (and indeed that's kind of the point!), taken on its own.
Even more confusing is the fact that the concept of "partial signature" is important in context, because we are usually talking about using adaptors within a multisig context, and it's very easy i think to be confused between partial signature and adaptor (signature). (And then even *more* confusing is the recent single-signer adaptor signature usage, as per Fournier's recent work - which is single signer ... but still used in multisig ...).
I kind of warmed to Fournier's otVES name. It's somehow a much clearer way to express what we are actually talking about. (paper: https://github.com/LLFourn/one-time-VES/blob/master/main.pdf ).

Reply:
heh you'll be happy to know that in https://eprint.iacr.org/2020/476.pdf  there is now additionally something called a "pre-signature" and an "adaptor" actually refers to the party that does the "adapting"!

I think it's fine if everyone sticks with adaptor signature as it's a rather well know term by now. otVES is a bit wonky. If I could do over I would just go with _one-time signature encryption_. I'm still working on how best to explain it. Whenever I try to explain it 1-on-1 I start of with "adaptor signatures are a kind of signature encryption" and go from there. That seems to work.